src/ClassDesignChallenge/Professors.java

```diff
@@ -0,0 +1,24 @@
+package ClassDesignChallenge;
+
+import java.time.LocalDate;
+
+public class Professors {
+
+    private String firstName;
+    private String lastName;
+    private char sex;
+    private LocalDate dateOfBirth;
+    private int age;
+    private long cnp;
```

Comment:
We need a long primitive for the cnp?

Reply:
The reason I have used long primitive for the cnp was that I have encountered an error, more exactly 

> Integer number too large.

Seems like if the number has 10 digits, or more exactly in the range -2147483647 -- 2147483647, it can be represented as an integer otherwise it wont.  Therefor I have used as a primitive the long type as it stores bigger values. Please let me know if there was another way to do this?! @AlexIliescu11

Reply:
Or should I go with a string?

Reply:
We can have a String data type for CNP. If we have an international application, we might have clients that have a social security number (and not CNP). That social security number can have numbers and letters.
Also, yes, an int couldn't hold a value so large as a CNP number.